app/src/main/java/com/blaizmiko/popcornapp/models/Movie/Collection.java

```diff
@@ -0,0 +1,46 @@
+package com.blaizmiko.popcornapp.models.Movie;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Collection {
```

Comment:
Collection of what? Exp -  im new developer and i see List\<Collection\> collections = ... Wat?